Update doc (#243)
logion-network/logion-api: website/docs/client/loc.md

draftRequest = await draftRequest.addMetadata({
    name: "Some name",
    value: "Some value"
});
```

```typescript title="deleteMetadata"
draftRequest = await draftRequest.deleteMetadata({
    name: "Some name"
});
```

### Submit

When done, the request must be submitted to the LLO for review:

```typescript
const pendingRequest = await draftRequest.submit();
```

### After review

If the LLO accepted your LOC, you can proceed and publish it:

```typescript
const acceptedRequest = await pendingRequest.refresh() as AcceptedRequest;
const openLoc = await acceptedRequest.open({ signer, autoPublish: true });
```

If you set `autoPublish` to false, you'll have to manually publish each item
manually after the call to open.

After everything has been published, just wait for the LLO to close the LOC:

```typescript
const closedLoc = await openLoc.refresh() as ClosedLoc;
```

## Transaction LOC

### Request

```typescript
const draftRequest = await locsState.requestTransactionLoc({
    legalOfficer: alice,
    description: "This is a Transaction LOC",
    draft: true,
    legalFee: Lgnt.from(2000),
});
```

:::warning
The default legal fee for a transaction

> LOC is 2000 LGNTs. Another value of `legalFee` should have been discussed with the LLO beforehands,
> otherwise it may reject the LOC. 

Reviewer: ```suggestion
LOC is 2000 LGNTs. Another value of `legalFee` should have been discussed with the LLO beforehand,
otherwise the LLO may reject the LOC. 
```

All possible transferred rights are available in the `logionLicenseItems` array.
* With the code `TIME` it is possible to limit the right in time by setting the parameter `expiration`.
* With the code `REG` it is possible to limit to some countries/regions with the parameter `regionalLimit`.
:::

## Tokens Records

A tokens record has attached files. The files are then stored in Logion's private IPFS network
ensuring their availability over time.

As for collection items, there are 2 possibilities when attaching files:
- immediate upload of the files on creation or
- creation followed by an explicit upload later on.

See the examples below.

```typescript title="Add record and provide file content"
const recordId = Hash.of("record-id");
const recordDescription = "Some tokens record";
const file = HashOrContent.fromContent(
    new NodeFile("integration/test.txt", "report.txt", MimeType.from("text/plain"))
);
closedLoc = await closedLoc.addTokensRecord({
    payload: {
        recordId,
        description: recordDescription,
        files: [ file ],
    },

>     signer: state.signer,

Reviewer: ```suggestion
    signer,
```

});
```

```typescript title="Add record and provide description only"
const recordId = Hash.of("record-id");
const recordDescription = "Some tokens record";
const file = HashOrContent.fromContentFinalized(
    new NodeFile("integration/test.txt", "report.txt", MimeType.from("text/plain"))
);
closedLoc = await closedLoc.addTokensRecord({
    payload: {
        recordId,
        description: recordDescription,
        files: [
            HashOrContent.fromDescription({
                name: file.name,
                hash: file.contentHash,
                size: file.size,
                mimeType: file.mimeType,
            }),
        ],

>     },

Reviewer: Add `signer` param:
```suggestion
    },
    signer,    
```
